billingWrapper/Intersections.Billing/BillingToken.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Intersections.Billing.Gateway;
+using Intersections.Billing.Gateway.Recurly;
+using Intersections.Billing.Model;
+using Intersections.Billing.Model.Recurly;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Intersections.Billing
+{
+    public class BillingToken
+    {
+        /// <summary>
+        /// Temporary constant until we have multiple billing providers
+        /// </summary>
+        private const string billingProvider = "recurly";
+
+        /// <summary>
+        /// Get Billing Token in exchange of Credit Card and Billing Information 
+        /// </summary>
+        /// <param name="tokenRequest">TokenRequest containing billing information</param>
+        /// <returns>TokenResponse</returns>
+        public async Task<TokenResponse> get(TokenRequest tokenRequest, string apiKey)
```

Comment:
are we expecting them to give us the token?  I would expect us to have the token directly in this wrapper so the end-client doesn't even have to worry about it.  I mean right now it's visible in the UI source anyway so it's basically public info at this point.

Reply:
They will give not give us the Token, rather they will provide us with TokenRequest consisting of Billing Address and Card Info etc. In return they will get the TokenResponse consisting of either the token or errors as the case may be.